autopush/metrics.py

```diff
@@ -71,12 +71,13 @@ def timing(self, name, duration, **kwargs):
 class DatadogMetrics(object):
     """DataDog Metric backend"""
     def __init__(self, api_key, app_key, flush_interval=10,
-                 namespace="autopush"):
+                 namespace="autopush", hostname="localhost"):
```

Comment:
hrm. tempted to say this should be None instead of "localhost". Settings sets it to a value, and that's pretty much the only init location.

That or we drop `get_hostname()` and treat the fact that it's been forced to None as an error.